bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/fields/DateTimeField.java

```diff
@@ -59,11 +60,11 @@ public static Control create(Composite composite, MField field, int row, int col
 			@Override
 			public List<String> getContent(String entry) {
 				ArrayList<String> result = new ArrayList<>();
-				Instant date = DateTimeUtil.getDateTime(Instant.now(), entry, locale);
+				Instant date = DateTimeUtil.getDateTime(LocalDateTime.now(ZoneId.of(timezone)).toInstant(ZoneOffset.UTC), entry, locale, timezone);
```

Comment:
Hier wird date- und timeUtil (also das Pattern) ignoriert?

`LocalDateTime.now(ZoneId.of(timezone)).toInstant(ZoneOffset.UTC)` das ist schon sehr komplex.
Soweit ich sehe erschließt sich das komplett aus der `timezone`, könnte man das nicht entsprechend in der `DateTimeUtil`-Methode machen, um Fehler bei der Nutzung zu vermeiden?

Reply:
Es wird schon gemacht, aber hier wurde einer der Methoden verwendet, die das Instant übergeben braucht.